src/ReleaseHistory.md

```diff
@@ -69,9 +69,13 @@
 * Rename annotatedCodeLocation.calleeKey member to 'targetKey'
 
 ## **v1.5.29** [Driver](https://www.nuget.org/packages/Sarif.Driver/1.5.29) | [SDK](https://www.nuget.org/packages/Sarif.Sdk/1.5.29)
-
 * Add `--quiet` option to suppress console output.
 
 ## **v1.5.30** [Driver](https://www.nuget.org/packages/Sarif.Driver/1.5.30) | [SDK](https://www.nuget.org/packages/Sarif.Sdk/1.5.30)
+* Add static helper method `AnalyzeCommandBase.LogToolNotification`.
+
+## **v1.5.31** [Driver](https://www.nuget.org/packages/Sarif.Driver/1.5.31) | [SDK](https://www.nuget.org/packages/Sarif.Sdk/1.5.31)
+* Minor updates to console output
```

Comment:
You'll want to take my merged change for 1.5.31. It's actually a breaking change.